fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -5119,7 +5123,7 @@ void testDeleteWhereSimple(IndexedType indexedType) throws Exception {
                     () -> recordStore.deleteRecordsWhere(SIMPLE_DOC, indexedType.isSynthetic() ? Query.field("simple").matches(qc) : qc));
             assertThat(err.getMessage(), containsString(indexedType.isSynthetic()
                     ? "deleteRecordsWhere not matching primary key " + SIMPLE_DOC
-                    : String.format("deleteRecordsWhere not supported by index %s", index.getName())));
+                    : "deleteRecordsWhere not supported by index " + index.getName()));
```

Comment:
The `index.getName()` seems like it should be a log info, not in the message.

Reply:
Maybe, yeah. That's controlled by the `RecordsWhereDeleter` in `FDBRecordStore`. I've left it as is in the name of not adding yet more to this PR, but we could reconsider